instrumentation/google.golang.org/grpc/otelgrpc/test/grpc_stats_handler_test.go

```diff
@@ -46,14 +47,18 @@ func TestStatsHandler(t *testing.T) {
 	serverMetricReader := metric.NewManualReader()
 	serverMP := metric.NewMeterProvider(metric.WithReader(serverMetricReader))
 
-	assert.NoError(t, doCalls(
+	listener := bufconn.Listen(bufSize)
+	defer listener.Close()
```

Comment:
I guess that as a follow-up PR we would like to use `net.Listen("tcp", "127.0.0.1:0")` here instead.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4532